readme.txt

```diff
@@ -14,22 +14,71 @@ A WordPress plugin to create block themes.
 
 This plugin allows you to:
 
-- Create a new theme, blank theme, child theme or style variation.
+- Create a blank theme
+- Create a new theme based on the currently active theme
+- Create a child theme of the active parent theme
+- Create a new style variation
+
+All newly created themes or style variations will include changes made within the WordPress Editor.
+
+This plugin also makes several changes to the contents of an exported theme, including:
+
+- Adds all images used in templates to the theme's `assets` folder.
+- Ensures the block markup used in templates and patterns is export-ready.
+- Ensures most strings used in templates and patterns are translate-ready.
 
 The plugin is development only — not intended for use on production websites, but used as a tool to create new themes.
 
 = Step 1 – Setup =
 Install and activate the [Create Block Theme](https://wordpress.org/plugins/create-block-theme) plugin.
 
-In the WordPress Admin Dashboard, under Appearance there will be a new page called:
+There will be a new panel accessible from the WordPress Editor, which you can open by clicking on a new icon to the right of the "Save" button, at the top of the Editor.
 
-- Create Block Theme
+1. Create Block Theme panel in the WordPress Editor
```

Comment:
This should embed screenshot-1.jpg with "Create Block Theme panel in the WordPress Editor" as the caption.